lib/src/components/NxSubmitMask/NxSubmitMask.tsx

```diff
@@ -28,9 +30,15 @@ const NxSubmitMask: FunctionComponent<Props> =
 
     return (
       <div className={classes}>
-        <h3 className="nx-h3 nx-submit-mask__message">
-          <NxLoadingSpinner>{spinnerMessage}</NxLoadingSpinner>
-        </h3>
+        <div className="nx-submit-mask__message">
+          { success ?
+            <>
+              <NxFontAwesomeIcon icon={faCheckCircle} className="nx-loading-spinner__icon" />
+              <span>{spinnerMessage || 'Loading…'}</span>
```

Comment:
This `|| 'Loading...'` is wrong.  This is the success message here.  At the same time, `spinnerMessage` will always have a truthy value here based on the way it's created above.

Reply:
> This `|| 'Loading...'` is wrong. This is the success message here. At the same time, `spinnerMessage` will always have a truthy value here based on the way it's created above.

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/7eaad8766f8001a37f8fff6f890d10ea7a8be8af